docs-user/guide-getting-started.md

```diff
@@ -0,0 +1,114 @@
+
+![A screenshot of the profiler popup](images/getting-started-popup.png)
+
+There you can change the preset as well as access the advanced settings.
+
+?> We
+suggest using the preset `Firefox` (also called `Nightly` if running in the
+nightly version of Firefox) if you want to profile the full browser (for
+example to [report a performance problem in Firefox](https://firefox-source-docs.mozilla.org/performance/reporting_a_performance_problem.html)),
+and `Web Developer` if you want to profile a web site.
+
+### Using the Devtools panel
+
+The same settings are available in the devtools panel.
+
+![A screenshot of the Devtools performance panel](images/getting-started-devtools-panel.png)
+
+!> As explained above, when opening the Devtools toolbox there's an
+overhead due to the other panels. We try that it stays minimal but you need to
+keep this into account when looking at profiles that were captured from this
+panel.
+
+## Capture a profile
+
+### Using the popup or the devtools panel
+
+[A video showing how to use the profiler popup](images/getting-started-use-popup.webm ':include :type=video controls')
+
+You can use the popup from the icon, or similarly the devtools panel, by
+clicking on `Start Recording`, then `Capture`. Then the profiler user interface
+will open to show the captured data.
+
+Note that the profiler user interface is an external web page, however it's
+cached locally the first time you access it. This means you'll need a network
+connection to download it the first time you use it. If you have a network
+connection in later uses and a new version is available, the UI will offer to
+switch to the new version.
+
+It's also important to know that the data stays local to your computer until you
+upload it (see below).
+
+### Using the profiler button
+
+It's possible to click on the button to start the profiler with current
+settings. Then click it again to capture the profile and the open the profiler
+UI.
+
+[A video showing how to use the profiler icon](images/getting-started-use-icon.webm ':include :type=video controls')
+
+### Using keyboard shortcuts
+
+The following keyboard shortcuts are available:
+
+* `Ctrl + Shift + 1`: start the profiler, or stop and discard the data if it's already running.
+* `Ctrl + Shift + 2`: capture the profile from the currently running session. It
+  has no effect if the profiler isn't running.
+
+## Share a profile
+
+![A screenshot highlighting the toolbar's upload profile button.](images/getting-started-upload.png)
+
+One of the most powerful and useful features of the Firefox Profiler is the ability to upload and share profiles. The first step is to hit the *Upload Local Profile* button. You'll be able to exclude some information before uploading. Then the profile will be uploaded to an online storage. This profile can then be shared in online chat, emails, and bug reports. Note that anybody that has the link will be access the uploaded data, as it's not protected otherwise.
```

Comment:
```suggestion
One of the most powerful and useful features of the Firefox Profiler is the ability to upload and share profiles. The first step is to hit the *Upload Local Profile* button. You'll be able to exclude some information before uploading. Then the profile will be uploaded to an online storage. This profile can then be shared in online chat, emails, and bug reports. Note that anybody that has the link will be able to access the uploaded data, as it's not protected otherwise.
```